client/src/utils/challenge-request-helpers.ts

```diff
@@ -0,0 +1,47 @@
+import { ChallengeFiles } from '../redux/prop-types';
+
+/*
+ * Express's body-parser has a default size limit of 102400 bytes for a request body.
+ * These helper functions make sure the request body isn't too big when saving or submitting multiFile cert projects
+ */
+
+export const MAX_BODY_SIZE = 102400;
+
+interface UniformizeRequestBodyArgs {
+  id: string;
+  challengeFiles: ChallengeFiles;
+  challengeType: number;
+}
+
+export function uniformizeRequestBody({
+  id,
+  challengeFiles = [],
+  challengeType
+}: UniformizeRequestBodyArgs) {
```

Comment:
Uniformize is a touch vague.  How about `transformRequestBodyForApi`?  It's all bad, but this seems slightly less bad.

Reply:
Uniformerizate?

Reply:
what about `standardizeRequestBody`?

Reply:
I like standardizeRequestBody :)